RStudio/machine-images/config/infra/provisioners/provision-rstudio.sh

```diff
@@ -19,6 +21,9 @@ sudo make
 sudo make install
 cd "../../.."
 
+# Cleanup R install tmp folder
+sudo rm -rf "/tmp/R"
```

Comment:
What happens if you don't rm these dirs from tmp and just let the OS take care of it?

Reply:
I don't know-I just moved it from below so it was closer to the code that adds that folder. It's quite possible that the system will clean it out before packaging into an AMI, but I guess it's fine to delete it just in case? Happy to discuss more.